RPLCD/gpio.py

```diff
@@ -156,12 +156,30 @@ def _set_backlight_enabled(self, value):
 
     # Low level commands
 
-    def _send(self, value, mode):
-        """Send the specified value to the display with automatic 4bit / 8bit
-        selection. The rs_mode is either ``RS_DATA`` or ``RS_INSTRUCTION``."""
+    def _send_data(self, value):
```

Comment:
Hm, I think in the case of the GPIO version, having a single version with a mode parameter might probably be better, since the method body is almost completely the same. Maybe you can re-create the `_send` method and call it from `_send_data` and `_send_instruction`?